code-dot-org/code-dot-org
# truncate audit log when copying levels

The `audit_log` field carries over when levels are copied into new version years. These lines become very long and make it hard to read diffs when the audit_log contents changes during normal level editing. 

This PR erases the audit_log when a level is copied. this should lead to easier-to-read diffs when levels are edited.

## Testing story

I tested the following locally:
* deep copy a script
* verify the audit_log of its levels has been truncated
* make an edit to one of the levels
* verify my email address now appears in the audit log
